python/ray/dashboard/subprocesses/handle.py

```diff
@@ -1,32 +1,22 @@
+from __future__ import annotations
```

Comment:
do we need this?

Reply:
Yes because I use the `|` operator for `Union` annotations operatorbelow. With this import, this operator can work in Python 3.9. Otherwise this operator only works after Python 3.10.

`headers: dict[str, str] | multidict.CIMultiDictProxy[str]`

Reply:
Lets use `Union` to be consistent with the remaining codebase